great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -61,33 +61,33 @@ class CloudMigrator:
     def __init__(
         self,
         context: BaseDataContext,
-        ge_cloud_base_url: Optional[str] = None,
-        ge_cloud_access_token: Optional[str] = None,
-        ge_cloud_organization_id: Optional[str] = None,
+        cloud_base_url: Optional[str] = None,
+        cloud_access_token: Optional[str] = None,
+        cloud_organization_id: Optional[str] = None,
```

Comment:
Don't think we need to deprecate since we've never really use this code path in prod?

Reply:
Agree